pkg/scorecard/statement.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Subject {
+// 	 Repo           jsonRepoV2          `json:"repo"`
+// }
+
+package scorecard
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	intoto "github.com/in-toto/attestation/go/v1"
+
+	docs "github.com/ossf/scorecard/v5/docs/checks"
+	sce "github.com/ossf/scorecard/v5/errors"
+)
+
+const (
+	PredicateType = "https://scorecard.dev/result/v1"
+)
```

Comment:
does `v1` imply predicate stability? Should this be v0.1 like in this comment?
https://github.com/ossf/scorecard/issues/3352#issuecomment-2368655593

Reply:
Yes, I agree. I switched it to 0.1 :+1: